src/morphAttrs.js

```diff
@@ -20,6 +20,9 @@ export default function morphAttrs(fromNode, toNode) {
             fromValue = fromNode.getAttributeNS(attrNamespaceURI, attrName);
 
             if (fromValue !== attrValue) {
+                if (attr.prefix === 'xmlns'){
+                    attrName = attr.name; // It's not allowed to set an attribute with the XMLNS namespace without specifying the `xmlns` prefix
```

Comment:
@eprev Don't we set `attrName = attr.name` above on line 14 already?  Or did it get wiped out by `attr.localName`?

Reply:
Yes and yes, it gets replaced with `attr.localName` on line 19.